docs/content/guides/dagster-pipes/modifying-existing-code-to-work-with-dagster-pipes.mdx

```diff
@@ -1,6 +1,270 @@
 ---
 title: "Modifying existing code to work with Dagster Pipes | Dagster Docs"
-description: ""
+description: "With Dagster Pipes, you can incorporate existing code into Dagster without huge refactors. This guide shows you how to modify existing code to work with Dagster Pipes."
 ---
 
 # Modifying existing code to work with Dagster Pipes
+
+In this guide, we’ll demonstrate how to modify existing code (e.g. an existing Python script) to work with [Dagster Pipes](/guides/dagster-pipes) and send metadata from the script to Dagster.
+
+<Note>
```

Comment:
visually, both of these Notes sections at the top of the page are a bit overwhelming. I think this first one could be fine not being a Note